src/main/kotlin/no/nav/helse/dokument/K9MellomlagringService.kt

```diff
@@ -19,12 +19,12 @@ class K9MellomlagringService(
     }
 
     internal suspend fun slettDokumeter(
-        urlBolks: List<List<URI>>,
+        vedleggIdBolks: List<List<String>>,
```

Comment:
Fint om du kaller det for dokumentIdBolks. Det er ikke bare vedlegg vi oppretter eller slettet :)